shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -831,7 +832,7 @@ - (void)dealloc {
   [self removeInternalPlugins];
   [self deregisterNotifications];
 
-  [_displayLink release];
+  [_keyboardAnimationVsyncClient release];
```

Comment:
Set the property to nil after it is released.
```objc
[_keyboardAnimationVsyncClient release];
_keyboardAnimationVsyncClient = nil;
```

Reply:
Done.